change/@fluentui-react-charting-7ed503a2-d9d1-4679-8074-c67eb83161e1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fixed accessibility issues in pie chart",
+  "packageName": "@fluentui/react-charting",
+  "email": "yushsingla@microsoft.com",
```

Comment:
Lets capture a screenshot before and after this change for default props and compare the diff.

Reply:
I'll drop the results here in the comment

Reply:
Dark Mode :
![image](https://github.com/microsoft/fluentui/assets/70366079/e81f711e-e180-4cf6-8245-b4e457c11338)

Default : 
![image](https://github.com/microsoft/fluentui/assets/70366079/29b48740-d4b2-4277-82a3-17e5fad4f478)

RTL: 
![image](https://github.com/microsoft/fluentui/assets/70366079/3cbffa9b-f5aa-4f78-a8a9-ede981a3426c)

I see the RTL labels are buggy, will work on fixing that

Reply:
lmk any other findings based on these differences

Reply:
Default screenshot is also showing in dark mode only.
If the user defined width and height are fixed, does the radius shrink to account for the text border circle.

Reply:
there was some mistake in uploading image, the default is corrected now

Reply:
If there's user defined width and height, we will subtract some value from that to accommodate the text. Such that the text doesn't overflow. 

![image](https://github.com/microsoft/fluentui/assets/70366079/e2fc5cae-0775-4e3c-a2fd-4939989fe681)